### ISSUE TITLE
Choice of Licensing

### ISSUE THREAD

**Issue by hendursaga** [Ludeme/Ludii#21]
It's been a long time since I have seen code licensed as Creative Commons, much less Non-Commercial, much less No Derivatives. What was the reasoning behind the licensing? This basically means one cannot even fork Ludii, yes? The closest forum post I could find is [#150](https://ludii.games/forums/showthread.php?tid=150), but that still doesn't answer my question.

**Comment by DennisSoemers** [comment]
Yeah, it wouldn't have been my first choice either personally. Technically, forking is probably still allowed as per the GitHub Terms of Agreement (which automatically allow creating forks of any repo with public visibility), but then changing anything in that fork might not be.

If you're curious about the reasoning behind the license, you could try sending a mail to Cameron Browne (I don't think he's actively keeping an eye on issues posted here).
